audbackend/core/api.py

```diff
@@ -82,43 +157,27 @@ def create(
         backend object
 
     Raises:
-        BackendError: if an error is raised on the backend
+        BackendError: if an error is raised on the backend,
+            e.g. repository cannot be created
         ValueError: if no backend class with alias ``name``
             has been registered
 
     Examples:
-        >>> create('file-system', 'host', 'doctest')
-        ('audbackend.core.filesystem.FileSystem', 'host', 'doctest')
+        >>> create('file-system', 'host', 'repository')
+        ('audbackend.core.filesystem.FileSystem', 'host', 'repository')
 
     """
-    if name not in backend_registry:
-        raise ValueError(
-            'A backend class with name '
-            f"'{name} "
-            'does not exist.'
-            "Use 'register_backend()' to register one."
-        )
-
-    if name not in backends:
-        backends[name] = {}
-    if host not in backends[name]:
-        backends[name][host] = {}
-    if repository not in backends[name][host]:
-        backend = backend_registry[name]
-        backends[name][host][repository] = utils.call_function_on_backend(
-            backend,
-            host,
-            repository,
-        )
-    return backends[name][host][repository]
+    backend = _backend(name, host, repository)
+    utils.call_function_on_backend(backend._create)
+    return backend
 
 
 def delete(
         name: str,
         host: str,
         repository: str,
 ):
-    r"""Delete repository and remove backend instance.
+    r"""Delete repository.
```

Comment:
What did "remove backend instance" referred to?

Reply:
It's explained in the docstring, so I don't think we need to mention it here.

Reply:
![image](https://user-images.githubusercontent.com/10383417/233608567-ef9bf5b3-2d2f-4f3d-9335-92305bbdc158.png)

The caption is now aligned with that of the other functions.

Reply:
Thanks.